packages/atomic/src/components/commerce/atomic-commerce-interface/atomic-commerce-interface.ts

```diff
@@ -130,7 +130,7 @@ export class AtomicCommerceInterface
   /**
    * The commerce interface i18next instance.
    */
-  @property({type: Object}) i18n: i18n;
+  @property({type: Object, attribute: false}) i18n: i18n;
```

Comment:
Small addition: this disallows setting i18n by attribute (but it remains a reactive property).
It didn't make sense for those property to be attribute.